CHANGELOG.md

```diff
@@ -46,20 +50,16 @@ This repository contains the `FreeRTOS AWS Reference Integrations`, which are pr
 - Added FreeRTOS Console API to interact with CLI over common IO or UDP interface.
 - Added sample which demonstrates executing commands using FreeRTOS+CLI and UART interface.
 
-#### FreeRTOS+TCP V2.3.1
+#### FreeRTOS+TCP V2.4.0
 
 - Sub-moduled to the [FreeRTOS/FreeRTOS-Plus-TCP](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP) repository
```

Comment:
See the modified text below.

- Sub-moduled to the [FreeRTOS/FreeRTOS-Plus-TCP](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP) repository. This is a breaking change for customer using `202007` release of amazon-freertos due to change in folder structure. The version bump was done in the single source of truth of FreeRTOS+TCP (`V2.2.2` -> `V2.3.0`). Since the version number in amazon-freertos is already at `2.3.0` (discrepancy due to multiple sources of truth), it is receiving a minor version bump to `2.3.1`. 

@gitwallit, any comments? I tried to make this descriptive for clarity.

Reply:
single source of truth may not be familiar with other viewers. we could perhaps say - "The version bump for this change was done in the FreeRTOS-Plus-TCP repository (V2.2.2 -> V2.3.0). Since the version number in the amazon-freertos repository is already at 2.3.0 (this discrepancy is caused by the presence of multiple FreeRTOS+TCP sources, which has since been corrected with the FreeRTOS-Plus-TCP repository as the single source of truth), only a minor version bump to 2.3.1 is being done.

Reply:
In essence, the whole thing should read:

"Sub-moduled to the [FreeRTOS/FreeRTOS-Plus-TCP](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP) repository. This is a breaking change for users using `202007` release of amazon-freertos due to change in folder structure. The version bump for this change was done in the FreeRTOS-Plus-TCP repository (V2.2.2 -> V2.3.0). Since the version number in the amazon-freertos repository is already at 2.3.0 (this discrepancy is caused by the presence of multiple FreeRTOS+TCP sources, which has since been corrected with the FreeRTOS-Plus-TCP repository as the single source of truth), only a minor version bump to 2.3.1 is being done."

Is this accurate?